pages/profile.js

```diff
@@ -0,0 +1,88 @@
+import Header from "@/components/Header";
+import {
+  ErrorMessageNotSignedIn,
+  HeaderArea,
+  LoginDataContainer,
+  ProfileImage,
+  UserGreeting,
+  HeadingBookmarkedActivities,
+  Ul,
+  Li,
+  LinkDetailsPage,
+  ErrorMessage,
+} from "@/components/styledComponents/profile.styled";
+import Bookmark from "@/components/Bookmark";
+import { BookmarkContainer } from "@/components/styledComponents/activityList.styles";
+import ActivityCard from "@/components/ActivityCard";
+import Login from "@/components/Login";
+import { useSession } from "next-auth/react";
+
+export default function ProfilePage({
+  onHandleBookmark,
+  activities = [],
+  bookmarkedActivities = [],
+}) {
+  const bookmarkedActivitiesDetails = activities.filter((activity) =>
+    bookmarkedActivities.includes(activity._id)
+  );
+
+  const { data: session } = useSession();
+
+  if (!session || !session.user) {
+    return (
+      <>
+        <ErrorMessageNotSignedIn>
+          Sie sind nicht eingeloggt <br /> - bitte einloggen!
```

Comment:
Is your app in english or german? 🙂

Reply:
You're absolutely right! My bad! I was so happy that the code worked that I completely forgot to write this in English! Thanks for the hint! ☺️